.github/workflows/playwright.yml

```
@@ -38,11 +45,24 @@ jobs:
       - name: Upload Playwright report
         uses: actions/upload-artifact@v4
         if: always()
         with:
           name: playwright-report
-          path: playwrightTests/playwright-report/
+          path: ./playwrightTests/playwright-report/
           retention-days: 1
+
+      - name: Deploy report to GitHub Pages
+        if: always()
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.4)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)